packages/odata-common/src/request-builder/get-all-request-builder-base.ts

```diff
@@ -64,11 +64,38 @@ export abstract class GetAllRequestBuilderBase<
    * @param orderBy - OrderBy statements to order the response by
    * @returns The request builder itself, to facilitate method chaining
    */
-  orderBy(orderBy: Orderable<EntityT>[]): this;
-  orderBy(...orderBy: Orderable<EntityT>[]): this;
   orderBy(
-    first: undefined | Orderable<EntityT> | Orderable<EntityT>[],
-    ...rest: Orderable<EntityT>[]
+    orderBy: Orderable<
+      EntityT,
+      DeSerializersT,
+      EntityApi<EntityBase, DeSerializersT>
```

Comment:
I wonder about the unspecific `EntityApi<EntityBase, DeSerializersT>`, if we don't specify it here, it could be the default, couldn't it?